be/src/Project/Services/Ordering/Ordering.Api/Infrastructure/Orders/Entities/OrderAttachment.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Foundation.GenericRepository.Entities;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Ordering.Api.Infrastructure.Orders.Entities
+{
+    public class OrderAttachment : EntityMedia
+    {
+        [Required]
+        public Guid OrderId { get; set; }
+
+        internal IEnumerable<Guid> Select(Func<object, object> p)
```

Comment:
**suggeston:** to remove

Reply:
@dawiddworak88 Ups, I haven't seen it.

Now is fixed